appinventor/components/src/com/google/appinventor/components/runtime/ListView.java

```diff
@@ -190,7 +190,7 @@ public void Width(int width) {
   @DesignerProperty(editorType = PropertyTypeConstants.PROPERTY_TYPE_BOOLEAN,
       defaultValue = DEFAULT_ENABLED ? "True" : "False")
   @SimpleProperty(description = "Sets visibility of ShowFilterBar. True will show the bar, " +
-      "False will hide it.")
+      "false will hide it.")
```

Comment:
this is also a noun; it the value **False** with capital F that needs to be specified; as shown in line 191.